DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\AclBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * This class contains the configuration information.
+ *
+ * @author Johannes M. Schmitt <schmittjoh@gmail.com>
+ */
+class Configuration implements ConfigurationInterface
+{
+    /**
+     * Generates the configuration tree builder.
+     *
+     * @return TreeBuilder The tree builder
+     */
+    public function getConfigTreeBuilder()
+    {
+        $tb = new TreeBuilder();
+        $rootNode = $tb->root('acl');
+
+        $rootNode
+            ->children()
+                ->scalarNode('connection')
+                    ->defaultNull()
+                    ->info('any name configured in doctrine.dbal section')
+                ->end()
+                ->arrayNode('cache')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('id')->end()
+                        ->scalarNode('prefix')->defaultValue('sf2_acl_')->end()
```

Comment:
Is it possible to rename `sf2_acl_` to `sf_acl_` or would that change break BC?

Reply:
I don't think this will break anything to be honest, unless you want your cache to persist between upgrades.